sdk/cognitiveservices/cognitiveservices-formrecognizer/README.md

```diff
@@ -2,11 +2,15 @@
 
 This package contains an isomorphic SDK for FormRecognizerClient.
 
+> Please note, a newer package [@azure/ai-form-recognizer](https://www.npmjs.com/package/@azure/ai-form-recognizer) is available as of August, 2020. While this package will continue to receive critical bug fixes, we strongly encourage you to upgrade.
```

Comment:
```suggestion
> Please note, this package is deprecated, and as of August 2020 the Form Recognizer SDK for JavaScript has moved to [@azure/ai-form-recognizer](https://www.npmjs.com/package/@azure/ai-form-recognizer). This deprecated package uses a version of the Form Recognizer service (`v1.0-preview`) that is no longer available, and therefore **is not functional**. This package will continue to receive critical security bug fixes, but it can no longer be used with the public Azure Form Recognizer service, and you must upgrade to `@azure/ai-form-recognizer` to use the service.
```

What do you think?

Reply:
If it is not functional, there is no point of providing security bug fixes either right?

Reply:
@ramya-rao-a I have no strong feelings about it. One could argue that if for whatever reason we depended on something with the right kind of very bad security vulnerability (like something that grants privilege escalation during installation), we should try to fix that even though this package doesn't do anything. I don't think that's likely at all, and we almost definitely wouldn't do anything that affects the runtime behavior of the code (since that doesn't exist anymore).

Reply:
alright, going with the below

> Please note, this package is deprecated and not functional as it uses a version of the Form Recognizer service (`v1.0-preview`) that is no longer available. As of August 2020, you can instead use [@azure/ai-form-recognizer](https://www.npmjs.com/package/@azure/ai-form-recognizer).